addons/account/tests/test_account_payment.py

```diff
@@ -933,6 +935,40 @@ def test_payment_name(self):
         payment.action_post()
         self.assertRegex(payment.name, r'BNK1/\d{4}/00002')
 
+    def test_payment_method_sequence_name(self):
+        AccountPayment = self.env['account.payment']
+        AccountPayment.search([]).unlink()
```

Comment:
This might be an issue depending on which modules are installed if they add constraints.
It's usually safer to start a new journal for instance